data/merge_acs_data.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+import numpy as np
+from pandas.core.reshape.merge import merge
```

Comment:
remove this and numpy, as they're unused (fyi vscode lightens the font on unused imports)